docs/usage_guide.rst

```diff
@@ -198,6 +198,39 @@ By choosing the "user & security" section of the settings you can change the use
    :alt: Changing the user's password
    :align: center
 
+Customizing the DeFi settings
+====================================
+
+By choosing the "Defi" section of the settings you can customize the DeFi settings of the application.
+
+The benefit of enabling only the modules you use, and specifying the addresses, is that rotki will only
+query only the specified addresses for the enabled modules. This can improve the query times by a lot.
```

Comment:
This can improve the query times by a lot -> This can considerably improve the querying speed

Reply:
Btw this would be a good way to use the suggestion feature of github :)